src/NumberTheory/NmbThy.jl

```diff
@@ -85,7 +85,7 @@ end
 
 norm_equation(R::Hecke.NfRelOrd{nf_elem,Hecke.NfOrdFracIdl}, a::NfAbsOrdElem{AnticNumberField,nf_elem}) = map(x -> R(evaluate(x)), norm_equation_fac_elem(R, a))
 
-function is_irreducible(a::NfAbsOrdElem{AnticNumberField,nf_elem})
+function is_irreducible(a::NfAbsOrdElem{AnticNumberField,nf_elem}) # TODO: move to Hecke
```

Comment:
```suggestion
function is_irreducible(a::NfAbsOrdElem{AnticNumberField,nf_elem})
```
Not possible

Reply:
Ah, because of `solve_mixed`, I missed that. OK, then we a comment should be added to explain why need to commit this act of type piracy :-)

Reply:
"Type piracy" aka using julia as intended originally (extending methods).